### DISCUSSION TITLE
desktopfolder and elementary os

### DISCUSSION THREAD

**Discussion by SystemError12** [elementary/triage#641] [Q&A]
Hi there, first time writing here. I am a big fan of elementary os and have it installed on my secondary system. My mainsystem is still windows, but who knows I might change one day ;)

I noticed an issue with [desktopfolder](https://github.com/calamityjoe87/desktopfolder) and elementary os version 8. It does install fine and lets me add desktop icons, folders, etc. Same as it did before on 7.1. However with the desktopfolder-app installed I can*t use other apps in full-screen. I tested particularly VLC and gwenview. My guess it's probably not only those 2 apps. Anyone have a similar experience and maybe knows a solution to this? 
For now all I could do is uninstall the desktopfolder again with a sad eye.. . 

Anyways keep up the great work. :)

**Comment by teamcons** [comment]
Hi

That would be an issue with Desktopfolder. Looking at it it did not have an update in one year
It needs to be maintained to keep working with the different new versions of elementary OS

Particularly, it works with Classic version ("X11", the current default), but needs also to be updated to work for the Secure session ("Wayland")
They have this note:
> Wayland (in Ubuntu since 17.10) is not fully supported yet, and we've experienced problems with adding support for it. We are working hard to fix this. You're welcome to try Desktop Folder and help us find or fix issues.

It also needs to be moved on to Flatpak, as the rest of the ecosystem is moving distribution to this format
If you poke at them, donate and encourage donating to them, maybe the developers of Desktopfolder would consider it :)

Much more basic/less eOS-y, there is XFCE [xfdesktop4 ](https://docs.xfce.org/xfce/xfdesktop/start)
It should work on Classic. Some development version have support for Secure session.
